#! /bin/sh /usr/share/dpatch/dpatch-run
## 08_mythmusic_ubuntu_directories.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Change some paths to match Debian/Ubuntu better

@DPATCH@

Index: mythmusic/globalsettings.cpp
===================================================================
--- a/mythmusic/mythmusic/globalsettings.cpp	(revision 13505)
+++ b/mythmusic/mythmusic/globalsettings.cpp	(working copy)
@@ -30,7 +30,7 @@
 #ifdef Q_WS_MACX
     gc->setValue(QDir::homeDirPath() + "/Music");
 #else
-    gc->setValue("/mnt/store/music/");
+    gc->setValue("/var/lib/mythtv/music/");
 #endif
     gc->setHelpText(QObject::tr("This directory must exist, and the user "
                     "running MythMusic needs to have write permission "
